app/views/course/leaderboards/show.html.slim

```diff
@@ -1 +1,43 @@
-p To be implemented later
+= page_header t('course.leaderboard.index.header')
```

Comment:
I had to be explicit here because this is `leaderboards` controller, so the key is `course.leaderboards.index.header` if we didn't state the translation key. 

Just thought that it will be easier to translate it to `leaderboard` instead of `leaderboards. Same for the comment below.